packages/gatsby/src/bootstrap/load-plugins/index.ts

```diff
@@ -82,6 +82,7 @@ const normalizeConfig = (config: IRawSiteConfig = {}): ISiteConfig => {
 }
 
 export async function loadPlugins(
+  reporter: Reporter,
```

Comment:
I think this should be the last argument, and optional. The function that you're passing it to has it as optional.

Reply:
There's only one module using this function, I suggest to leave it as mandatory